### DISCUSSION TITLE
Fixed: Unable to access online console (web interface)

### DISCUSSION THREAD

**Discussion by 99isfailure** [Security-Onion-Solutions/securityonion#5814] [General] [answered]
I recently re-built my security onion machine (multiple times in attempt to fix this issue) with the official SO iso image 2.3.80. As the title suggests, I can't get into the console online. When I navigate to the IP I set for the host, the page give a message of "Access to [IP Add.] was denied" - "You don't have the user rights to view this page." - "HTTP ERROR 403". I'm using chrome V. 94.0.4606.81 (I've also tried edge). I run so-allow - analyst - [local ip range] and again with my main workstation (static) ip.

Tech Info:
HP Z210 Workstation running VMWare Workstation Pro 16.1.2, SO VM Specs: Centos 7 64bit, 350GB HD, 4 Processors, 13.5GB RAM, and 2 Bridged network adapters.

**Comment by dougburks** [comment, marked as answer]
What options are you choosing during Setup?

Have you tried following the steps in this guide?
https://docs.securityonion.net/en/2.3/first-time-users.html

**Reply by 99isfailure** [reply]
I'm not having this issue anymore (my reasoning for changing the title to fixed). I tried deleting this post but I can't figure it out.

**Reply by Noby1Shin0b1** [reply]
Are you able to share your solution?
